app/poros/login_handler.rb

```diff
@@ -0,0 +1,8 @@
+class LoginHandler
+  class << self
```

Comment:
this class is here to build the session and to slim down large controller actions